qpext/qpext.Dockerfile

```diff
@@ -1,15 +1,18 @@
 # Build the inference qpext binary
 FROM golang:1.20 as builder
 
+# To stamp vcs info on the binary
+COPY .git .git
```

Comment:
To avoid this log message
```
{"severity":"INFO","timestamp":"2023-11-25T05:33:33.352137953Z","caller":"logging/config.go:80","message":"Unable to read vcs.revision from binary"}
```

Reply:
this could be a lot adding to the image?

Reply:
Reverting the change as we are unable modify the vcs.revision without adding .git folder